src/pages/Food/FoodModal/index.jsx

```diff
@@ -1,32 +1,64 @@
 import { useState } from 'react';
+import PropTypes from 'prop-types';
 import MainButton from 'components/MainButton';
 import SFoodModal from './style';
 
-function FoodModal() {
+function FoodModal({ menuElement }) {
   const [renderModal, setRenderModal] = useState(true);
 
   const hideTheModal = () => {
     setRenderModal(!renderModal);
```

Comment:
So, if it's true and I call it twice...
I'll be calling "hideTheModel" to show it ? :( 
What you did there is a `toggleModal`

Reply:
I chose these names because when i click on "makeTheModalAppear" the modal takes all place on my page. It can not be possible to click on this function twice because the modal is above the button. The only way to remove it is by clicking on "hideTheModal" function in the foodModal component.

Reply:
In your website maybe, but the function name is still misleading, that's not a good idea